src/pudl/metadata/fields.py

```diff
@@ -620,6 +616,15 @@
         "type": "integer",
         "description": "Year boiler was or is expected to be in compliance with federal, state and/or local regulations for sulfur dioxide emissions.",
     },
+    "cooling_id_eia": {
+        "description": (
+            "Identification Code for Cooling System (if multiple cooling "
```

Comment:
Can we normalize the capitalization here to just say "Identification code reporting to EIA for cooling system (if multiple...."